### PR TITLE
Error on deprecated

### PR DESCRIPTION
**Pull request by peter-lawrey**
This needs the PR https://github.com/OpenHFT/Chronicle-Core/pull/637 to pass

### CODE REVIEW

File: src/test/java/net/openhft/chronicle/bytes/internal/UnsafeTextTest.java [hunk 1]
```diff
@@ -0,0 +1,235 @@
+/*
```
Comment (tgd) [inline src/test/java/net/openhft/chronicle/bytes/internal/UnsafeTextTest.java:1]: Added by accident?

File: src/main/java/net/openhft/chronicle/bytes/internal/UnsafeText.java [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+import net.openhft.chronicle.core.*;
+
+import static net.openhft.chronicle.core.UnsafeMemory.MEMORY;
+import static net.openhft.chronicle.core.UnsafeMemory.UNSAFE;
+
+/**
+ * These are fast, unsafe ways to render text.
+ * NOTE: The caller has to ensure there is always plenty of memory to perform this operation.
+ */
+public final class UnsafeText {
```
Comment (tgd) [inline src/main/java/net/openhft/chronicle/bytes/internal/UnsafeText.java:29]: Hi Peter is this a brand new class or refactored from something else?

**Review by tgd — APPROVED** [review]
Hi Peter - approved it but with a few questions whether some unrelated changes were added

File: src/test/java/net/openhft/chronicle/bytes/internal/UnsafeTextTest.java (hunk 1 above)
Comment (peter-lawrey) [reply src/test/java/net/openhft/chronicle/bytes/internal/UnsafeTextTest.java:1]: The class this tests was moved from Core to Bytes.

File: src/main/java/net/openhft/chronicle/bytes/internal/UnsafeText.java (hunk 2 above)
Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/bytes/internal/UnsafeText.java:29]: Moved from Core to Bytes to make it internal to Bytes, and eventually phase it out.

File: pom.xml [hunk 3]
```diff
@@ -405,6 +406,15 @@
     </scm>
 
     <profiles>
+        <profile>
+            <id>java11-release8</id>
+            <activation>
+                <jdk>[11,</jdk>
```
Comment (nicktindall) [inline pom.xml]: This is not a valid range https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html

Should be `[11,)`

File: src/main/java/net/openhft/chronicle/bytes/BytesUtil.java [hunk 4]
```diff
@@ -131,6 +131,7 @@ static int[] isTriviallyCopyable0(@NotNull Class<?> clazz) {
      * @param fields The fields of the class.
      * @return An array of two integers, where the first element is the minimum offset, and the second element is the maximum offset.
      */
+    @SuppressWarnings("deprecation")
```
Comment (nicktindall) [inline src/main/java/net/openhft/chronicle/bytes/BytesUtil.java]: We shouldn't be suppressing deprecation warnings, that will make these things very hard to find down the track. Unless the method itself is due to go at the same time as the deprecated thing it references?

File: src/main/java/net/openhft/chronicle/bytes/internal/ByteBuffers.java [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+import java.nio.ByteBuffer;
+
+/**
+ * Utility class for working with {@link ByteBuffer} instances.
+ * <p>
+ * Provides functionality to directly set the address and capacity of a ByteBuffer.
+ * <p>
+ * Note: This class is deprecated and will be removed in version x.26.
+ */
+public final class ByteBuffers {
```
Comment (nicktindall) [inline src/main/java/net/openhft/chronicle/bytes/internal/ByteBuffers.java:31]: If this class is deprecated, can we add a deprecated annotation.

Also add `@deprecated` in the javadoc with a recommended alternative.

**Review by nicktindall — APPROVED** [review]
Approved, but we should make sure anything deprecated is marked/documented as such and be careful about suppressing deprecation warnings as it'll cause us headaches later on when we remove those things.

File: src/main/java/net/openhft/chronicle/bytes/internal/ByteBuffers.java (hunk 5 above)
Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/bytes/internal/ByteBuffers.java:31]: The original was deprecated, but not this one.

File: src/main/java/net/openhft/chronicle/bytes/BytesUtil.java (hunk 4 above)
Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/bytes/BytesUtil.java]: I have narrowed the SuppressWarning tot he field that needs it

**Pull request merged**